.github/workflows/release.yml

```diff
@@ -53,6 +53,7 @@ jobs:
         uses: armory-io/plugin-metadata-updater@master
         env:
           GITHUB_OAUTH: ${{ secrets.GH_PAT }}
+          JAVA_HOME: ""
```

Comment:
This does fix the issue with the action. Thanks to @danielpeach for suggesting I try this.

Reply:
Nice